src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,14 @@
-const humanCatDogYears = number => {};
-
+const humanCatDogYears = number => {
+    
+    if (number === 1) {
```

Comment:
you're missing the logic... what if I wanted to see the calculation when number is 4,5,6,7?